torchaudio/functional.py

```diff
@@ -75,6 +76,8 @@ def spectrogram(
         ``n_fft // 2 + 1`` and ``n_fft`` is the number of
         Fourier bins, and time is the number of window hops (n_frame).
     """
+    if power is not None and return_complex:
```

Comment:
based on the discussion below, we should get rid of this

Reply:
Okay. So the suggestion is that if power is provided, we return real-type tensor regardless of the `return_complex` value?